test/op_shape_test.cpp

```diff
@@ -346,6 +346,15 @@ TEST_CASE(broadcast)
                      migraphx::op::broadcast{0, lens},
                      input);
     }
+
+    {
+        std::vector<std::size_t> lens{1, 1};
+        migraphx::shape input{migraphx::shape::float_type, {1}, {0}};
+        expect_shape(migraphx::shape{migraphx::shape::float_type, {1, 1}, {0, 0}},
+                     migraphx::op::broadcast{1, lens},
+                     input);
+    }
+
     {
         std::vector<std::size_t> lens{1, 1};
         migraphx::shape input{migraphx::shape::float_type, {4, 1, 3}};
```

Comment:
This test should be removed as it fails, since it no longer throws.

Reply:
removed